cumulusci/tasks/bulkdata/load.py

```diff
@@ -135,11 +137,13 @@ def _run_task(self):
                             raise BulkDataException(
                                 f"Step {after_name} did not complete successfully: {','.join(result.job_errors)}"
                             )
+                results[name] = result
         if self.options["set_recently_viewed"]:
             try:
                 self._set_viewed()
             except Exception as e:
                 self.logger.warning(f"Could not set recently viewed because {e}")
+        self.return_values = {"step_results": results}
```

Comment:
@prescod Returning this info in the return_values is good, but we should probably make sure that it gets serialized with meaningful keys rather than just a tuple.

Reply:
@davisagli What serializer do we use and when? Is this related to freezing for MetaDeploy? Or Tubular API? JSON or YAML? Or both?

Reply:
a7af5aab7adb77

Reply:
It's not related to freezing steps for MetaDeploy. Both MetaCI (https://github.com/SFDO-Tooling/MetaCI/blob/main/metaci/build/flows.py#L33) and tubular (https://github.com/SFDO-Tooling/tubular/blob/main/tubular/worker/runner.py#L96) serialize return_values to JSON.

Would it make sense to switch from the namedtuple to a pydantic model so you can just call `.json()` rather than a custom serializer function? I'll approve anyway...